docs/how-to/monitor/update-metrics.md

```diff
@@ -31,5 +31,7 @@ The following table lists the name changes for Teku metrics:
 | `storage_finalized_state_branch_nodes`         | `storage_finalized_state_branch_nodes_total`         |
 | `storage_finalized_state_leaf_nodes`           | `storage_finalized_state_leaf_nodes_total`           |
 | `storage_finalized_states_stored`              | `storage_finalized_states_stored_total`              |
+| `validator_attestation_publication_delay`      | `validator_attestation_publication_delay_total`      |
+| `validator_block_publication_delay`            | `validator_block_publication_delay_total`            |
```

Comment:
Q: do we need `block_import_delay_counter_total` somewhere in this table as well?

Reply:
Correct, added now. Thanks for catching this.